Services/UI/OJS.Services.Ui.Business/Implementations/SubmissionsBusinessService.cs

```diff
@@ -406,6 +406,17 @@ is ExecutionStrategyType.NotFound
             return;
         }
 
+        if (submissionType.ExecutionStrategyType
+            is ExecutionStrategyType.RunSpaAndExecuteMochaTestsExecutionStrategy
+            or ExecutionStrategyType.RunSpaAndExecuteMochaTestsExecutionStrategySeparateTests
+            or ExecutionStrategyType.NodeJsV20RunSpaAndExecuteMochaTestsExecutionStrategySeparateTests)
+        {
+            problem.AdditionalFiles = await this.problemsDataService
+                .GetByIdQuery(model.ProblemId)
+                .Select(p => p.AdditionalFiles)
+                .FirstAsync();
+        }
```

Comment:
Why not load the `AdditionalFiles` with the Problem itslef?

Reply:
The problem is retrieved from the `cache`, if we store the `additional files` in the cache, considering that the additional files are mostly large files, this would lead to `bloating` of the cache.